src/autolabel/tasks/multilabel_classification.py

```diff
@@ -0,0 +1,200 @@
+from collections import defaultdict
+from typing import List, Dict, Tuple
+
+from langchain.prompts.prompt import PromptTemplate
+from sklearn.metrics import accuracy_score
+
+from autolabel.confidence import ConfidenceCalculator
+from autolabel.configs import AutolabelConfig
+from autolabel.schema import LLMAnnotation, Metric, MetricResult
+from autolabel.tasks import BaseTask
+from autolabel.tasks.utils import compute_f1
```

Comment:
I think compute_f1 requires some changes as a part of this PR: https://github.com/refuel-ai/autolabel/blob/107962437ab04f024b7942380b965a79f94a4f59/src/autolabel/tasks/utils.py#L25

That function currently splits string using .split() and the default separator. We can either change that function to take in a list of tokens that are pre-split or allow sending a custom separator. I believe this classes uses ',' as the separator between labels.

Reply:
Also, there we are calculating a micro F1 score. We can instead use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html to give the user the option to change between micro, macro, weighted, or class wise F1 scores.

Reply:
Since `QuestionAnsweringTask` uses this method as well, I'm just going to leave `compute_f1` as is for now and use `sklearn`'s `f1_score` directly in `MultilabelClassificationTask.eval`